docs/api/Document.md

```diff
@@ -15,6 +15,8 @@ A Sketch document.
 | id<span class="arg-type">string</span>             | The unique ID of the document. |
 | pages<span class="arg-type">[Page](#page)[]</span> | The pages of the document.     |
 | path<span class="arg-type">string</span>           | The path to the document (or the appcast URL in case of a Document from a remote Library.     |
+| colors<span class="arg-type">[GraientAsset]</span>           | A list of color assets defined in the document. Mutating the returned array will update the document colors.     |
```

Comment:
The documentation for the asset is missing

Reply:
I wasn't sure where it should go

Reply:
Same with the global getters

Reply:
So I'd create a new file in the `Models` section called `Asset`. Then each asset is a `##` header and the global getter a `###` under it.

What do you think?

Reply:
Seems like it makes sense. I haven't worked with the docs enough yet to picture what that would actually look like. I can try it. 

It's not totally clear in the Readme if I can build and view the docs locally. Is that possible?

Reply:
Ok, added an Assets section to the docs.